packages/components/src/components/textarea/textarea.stories.ts

```diff
@@ -30,400 +32,276 @@ export default {
-              { type: 'attribute', name: 'help-text', value: 'textarea must be filled' },
-              { type: 'attribute', name: 'form', value: 'testForm' },
-              { type: 'attribute', name: 'required', value: true },
-              { type: 'attribute', name: 'style-on-valid', value: true }
-            ],
-            args
-          })}
-        </div>
-        <div class="mb-2">
-          ${generateTemplate({
-            constants: [
-              { type: 'attribute', name: 'label', value: 'Min Length' },
-              { type: 'attribute', name: 'name', value: 'min length field' },
-              { type: 'attribute', name: 'placeholder', value: '^.{3,}$' },
-              { type: 'attribute', name: 'help-text', value: 'value must meet minlength' },
-              { type: 'attribute', name: 'form', value: 'testForm' },
-              { type: 'attribute', name: 'required', value: true },
-              { type: 'attribute', name: 'minlength', value: 3 },
-              { type: 'attribute', name: 'style-on-valid', value: true }
-            ],
-            args
-          })}
-        </div>
-        <div class="mb-2">
-          ${generateTemplate({
-            constants: [
-              { type: 'attribute', name: 'label', value: 'Max Length' },
-              { type: 'attribute', name: 'name', value: 'max length field' },
-              { type: 'attribute', name: 'placeholder', value: '^.{0,3}$' },
-              { type: 'attribute', name: 'help-text', value: 'value cannot exceed maxlength' },
-              { type: 'attribute', name: 'form', value: 'testForm' },
-              { type: 'attribute', name: 'required', value: true },
-              { type: 'attribute', name: 'maxlength', value: 3 },
-              { type: 'attribute', name: 'style-on-valid', value: true }
-            ],
-            args
-          })}
-        </div>
-        <sd-button type="submit">Submit</sd-button>
-      </form>
-      <script>
-        function handleSubmit(event) {
-          const form = document.querySelector('#testForm');
-          const sdTextarea = Array.from(document.querySelectorAll('sd-textarea'));
-
-          const isValid = sdTextarea => sdTextarea.checkValidity();
-
-          if (sdTextarea.every(isValid)) {
-            event.preventDefault(); // Prevent the default form submission behavior
-
-            const formData = new FormData(form);
-            const formValues = Object.fromEntries(formData);
-
-            alert('Form submitted successfully with the following values: ' + JSON.stringify(formValues, null, 2));
-          }
-        }
-
-        document.querySelector('#testForm').addEventListener('submit', handleSubmit);
-      </script>
-    `;
-  }
+export const Readonly = {
+  name: 'Read only',
+  render: () => html`
+    <div class="w-[500px]">
+      <sd-textarea
+        value="Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod."
```

Comment:
Read only example Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod.

Reply:
@coraliefeil Should the lorem text be placeholder?

Reply:
Pls enter;
"Read only example Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod."

Reply:
@Vahid1919 
![image](https://github.com/user-attachments/assets/1c092ffd-f18e-48d7-80b4-339894c1eb8c)